documentation/asciidoc/topics/proc_backing_up_cluster.adoc

```diff
@@ -0,0 +1,53 @@
+[id='backing_up_clusters-{context}']
+= Backing Up {brandname} Clusters
+Create backup files that stores the state of {brandname} clusters to persistent volume claims.
+
+.Prerequisites
+
+* Create a {brandname} cluster of {datagridservice} nodes.
+* Create at least one cache and add some entries.
```

Comment:
Yeah I was wondering if we should go with the full "content that the Cache Manager controls" or something. I'll take a stab at it but was trying to avoid being wordy and ambiguous.